config_hostinit.go

```diff
@@ -43,10 +46,16 @@ func (c *HostInitConfig) Set() error {
 	_, err := coll.UpdateOne(ctx, byId(c.SectionId()), bson.M{
 		"$set": bson.M{
 			"ssh_timeout_secs": c.SSHTimeoutSeconds,
```

Comment:
Many of these config subsections don't use `bsonutil.MustHaveTag` keys. 
Should we use them, though?

Reply:
I'm not sure. I followed the convention of the file. I think this is a thing we could go back and fix if we wanted to? @john-m-liu Any particular reason to leave `bsonutil.MustHaveTag` out here?

Reply:
It's the same reasoning as other places. Using it reduces the opportunity for programmer error but adds boilerplate/indirection